From fdd3feb37e36bec2ad75d76f8ac4d0273c5c0a91 Mon Sep 17 00:00:00 2001
Message-Id: <fdd3feb37e36bec2ad75d76f8ac4d0273c5c0a91.1651196606.git.geliang.tang@suse.com>
From: "David E. Box" <david.e.box@linux.intel.com>
Date: Tue, 26 Jan 2021 12:55:08 -0800
Subject: [PATCH] platform/x86: intel_pmt_crashlog: Add dependency on
 MFD_INTEL_PMT
Git-commit: fdd3feb37e36bec2ad75d76f8ac4d0273c5c0a91
Patch-mainline: v5.12-rc1
References: bsn#210
Modified-by-SEL: No

All devices that expose Intel Platform Monitoring Technology (PMT)
crashlog are currently owned by the intel_pmt MFD driver. Therefore make
the crashlog driver depend on the MFD driver for build.

Fixes: 5ef9998c96b0 ("platform/x86: Intel PMT Crashlog capability driver")
Signed-off-by: David E. Box <david.e.box@linux.intel.com>
Link: https://lore.kernel.org/r/20210126205508.30907-3-david.e.box@linux.intel.com
Signed-off-by: Hans de Goede <hdegoede@redhat.com>
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 drivers/platform/x86/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/platform/x86/Kconfig b/drivers/platform/x86/Kconfig
index c23040af770d..175ae4ae0028 100644
--- a/drivers/platform/x86/Kconfig
+++ b/drivers/platform/x86/Kconfig
@@ -1380,6 +1380,7 @@ config INTEL_PMT_TELEMETRY
 
 config INTEL_PMT_CRASHLOG
 	tristate "Intel Platform Monitoring Technology (PMT) Crashlog driver"
+	depends on MFD_INTEL_PMT
 	select INTEL_PMT_CLASS
 	help
 	  The Intel Platform Monitoring Technology (PMT) crashlog driver provides
-- 
2.34.1

